src/plugins/intel_gpu/src/graph/impls/ocl_v2/utils/fused_ops_jitter.cpp

```diff
@@ -968,7 +968,8 @@ JitConstants make_activation_jit_constants(const std::string& suffix,
         break;
     }
     case activation_func::softplus: {
-        jit.add(make_jit_constant(macro_def, log(exp(input) + one)));
+        const JitTerm input_f = out_dt == ov::element::f16 ? JitTerm{"(float)input"} : input;
```

Comment:
For consistency it is better to use `convert_float` for input and `convert_half` for result, in order not to affect the work of other calculations

Reply:
Agree and updated it accordingly.